lib/common/cmdline.c

```diff
@@ -46,6 +46,9 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts) {
         { "quiet", 'q', 0, G_OPTION_ARG_NONE, &(common_args->quiet),
           "Be less descriptive in output.",
           NULL },
+        { "quiet", 'Q', 0, G_OPTION_ARG_NONE, &(common_args->quiet),
+          "Be less descriptive in output.",
+          NULL },
```

Comment:
I just noticed both of these are problematic ... many tools use -q or -Q for other things. At least all tools are consistent with -V. Giving tools the ability to specify the short option (if any) for quiet would be ideal.

Long term, I would love to standardize our options across tools, but I don't know when that would become a possibility.

Reply:
Easy enough to add directly to stonith_admin.